### ISSUE TITLE
Fix "fatal: Not a git repository" error

### ISSUE THREAD

**Issue by ductiletoaster** [stylelint/stylelint#3910]
Details can be found here: https://github.com/GerHobbelt/jison/issues/43

Styleling version: `9.10.0` (9.9.0 seems to be fine)

> How are you running stylelint: CLI, PostCSS plugin, Node API?

As a CLI

> What actually happened (e.g. what warnings or errors did you get)?

Running the following git cmd caused this error.
```
+ git clean -fdx
fatal: Not a git repository: node_modules/@gerhobbelt/recast/../../../.git/modules/lib/modules/js/recast
```

**Comment by ductiletoaster** [comment]
@jeddy3 Correct it was part of our CI process.

**Comment by jeddy3** [comment]
> Correct it was part of our CI process.

Thanks for the clarification.

The problem should be fixed in `9.10.1`, which was just released.

**Comment by ductiletoaster** [comment]
Thank you!
